test/unit/http/preprocessors.js

```diff
@@ -301,7 +289,7 @@ describe('preprocessors', () => {
             )
           )).should.be.rejectedWith(Problem, { problemCode: 401.2 }));
 
-        it('should do nothing on cookie auth with incorrect session token for non-GET requests', () =>
+        it('should reject cookie auth with incorrect session token for non-GET requests', () =>
```

Comment:
Maybe we can remove the part of the test description about "for non-GET requests". We will now fail even GET requests.